lib/provider.coffee

```diff
@@ -22,21 +27,32 @@ class BibtexProvider
   second `@`, as this would become confusing.
   ###
   wordRegex: XRegExp('(?:^|[\\p{WhiteSpace}\\p{Punctuation}])@[\\p{Letter}\\p{Number}\._-]*')
-  constructor: ->
-    @bibtex = []
+  atom.deserializers.add(this)
```

Comment:
Can we move both this and the @deserialize definition to immediately before the constructor definition? There doesn't seem to be any reason to separate them, and placing them together as in the Atom docs example will make it easier to see what's happening.